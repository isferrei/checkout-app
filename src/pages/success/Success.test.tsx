import { render } from "@testing-library/react";
import Success from "./index";

describe("Success component", () => {
  test("renders correctly", () => {
    const { asFragment } = render(<Success />);
    expect(asFragment()).toMatchSnapshot();
  });
});
