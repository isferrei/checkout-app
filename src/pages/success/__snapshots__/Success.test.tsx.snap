// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center h-screen"
  >
    <h2>
      Failed to fetch data. Please try again later.
    </h2>
  </div>
</DocumentFragment>
`;
